sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4763,6 +4833,12 @@ public partial class ScaleInPolicy
         public bool? ForceDeletion { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.Compute.Models.VirtualMachineScaleSetScaleInRule> Rules { get { throw null; } }
     }
+    public partial class ScheduledEventsProfile
```

Comment:
This class name still looks a bit general. Could we come up with a proper prefix to specify this?

Reply:
this is a complex object there is no good way to explain what it is just by class name, if user wants to know more they will need to read documentation

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/computeRPCommon.json#L1721

Reply:
changed to ComputeScheduledEventsProfile